frontend/src/components/layout/PageWrapper/PageWrapper.tsx

```diff
@@ -22,8 +22,8 @@ const BaseContentWrapper: React.FC<BaseContentWrapperProps> = ({
   children,
 }) => {
   return (
-    <main className="w-full max-w-1800 mx-auto bg-[#fff] py-24">
-      <div className="w-full h-full px-24 md:px-32 xl:px-48">{children}</div>
+    <main className="w-full max-w-1800 mx-auto bg-[#fff] pt-24">
+      <div className="w-full h-full px-24 md:px-32 lg:px-88">{children}</div>
```

Comment:
In the future, let's refactor tailwind out whenever we are changing styles that use it

Reply:
For sure.  Because we were planning to create a ticket for this refactoring, I decided to leave it alone for now.